### DISCUSSION TITLE
RenderAnim renders black video

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#146] [! Welcome - readme first!]
Used to render animation, but now gives a video with a black screen instead
https://cables.gl/p/Kl9XpX

Can't find the reason of the problem

**Comment by steam0r** [comment]
hey, if you remove the "Performance" op between MainLoop and RenderAnim it works fine for me. Maybe also put a Sequence in there so the order of execution is clear. But i think "Performance" is the actual problem.

**Reply by bbbbbgdn** [reply]
Wow, such an easy fix! Thanks, Performance op was a betrayer!
